### PR TITLE
Replace Amber::Router::Params::URL_ENCODED_FORM with hardcoded string

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

**Comment by marksiemers** [comment]
Not sure that coupling to that constant is valuable - and it breaks when names change in the amber repo.

File: src/amber_spec.cr [hunk 1]
```diff
@@ -15,7 +15,7 @@ end
 {% for method in %w(get head post put patch delete) %}
   def {{method.id}}(path, headers : HTTP::Headers? = nil, body : String? = nil)
     request = HTTP::Request.new("{{method.id}}".upcase, path, headers, body )
-    request.headers["Content-Type"] = Amber::Router::Params::URL_ENCODED_FORM
+    request.headers["Content-Type"] = "application/x-www-form-urlencoded"
```
Comment (marksiemers) [inline src/amber_spec.cr]: Should this line only be here for `post` `put` and `patch`?

**Comment by marksiemers** [comment]
The CI is failing - it seems to think it is ruby - looking for a Rakefile.

Also, this change will likely cause a merge conflict with https://github.com/amberframework/amber-spec/pull/6#discussion-diff-148189520R8

If that gets merged in, I can rebase, and also maybe limit this to `post` `put` and `patch` verbs - we do want that change as well, correct?

**Comment by eliasjpr** [comment]
Well we need helper methods to test the other http verbs. Or there is a better way of generating this helper methods?

**Comment by marksiemers** [comment]
Taken care of in https://github.com/amberframework/amber-spec/pull/9

**Pull request closed without merging**